src/osemgrep/reporting/Summary_report.ml

```diff
@@ -15,8 +15,29 @@ module Out = Semgrep_output_v1_t
 (* Entry point *)
 (*****************************************************************************)
 
+exception FoundGitDir of Fpath.t
+
+let find_git_dir lst =
```

Comment:
I usually use `xs` rather than `lst` to indicate "unimportant list". `roots` would be even better

Reply:
is there some history with `xs` – like `i, j, k`  for loops as they they are common index term for matrices?

Reply:
I believe `xs` is the plural of `x`. I think it comes from Haskell. I used `l` for many years because that's they used in the OCaml standard library. Some people claim that some fonts make it hard to distinguish `l` from `1`.

Reply:
thanks @mjambon – I was reading it as extra-small